File: core.shared/Net/DTO/V1/Notification/NOT_AllPerimeter_Request.cs
using Core.Shared.Net.Enumerations.V1;

namespace Core.Shared.Net.DTO.V1.Notification
{
    public class NOT_AllPerimeter_Request: NOT_Perimeter_Request
    {
        public EnuNotificationType? Type { get; set; } = EnuNotificationType.Dashboard;
        public DateTime? DateTime { get; set; } = null;
        public string NumParticipant { get; set; } = string.Empty;
    }
}

File: core.shared/Typescript/DTO/V1/Notification/nOT_AllPerimeter_Request.ts
/* Auto Generated */

import { NOT_Perimeter_Request } from "./nOT_Perimeter_Request";
import { EnuNotificationType } from "./../../../Enumerations/V1/enuNotificationType";

export interface NOT_AllPerimeter_Request extends NOT_Perimeter_Request {
    type?: EnuNotificationType;
    dateTime?: Date;
    numParticipant: string;
}
